app/controllers/TaxYearSummaryController.scala

```diff
@@ -149,6 +149,13 @@ class TaxYearSummaryController @Inject()(taxYearSummaryView: TaxYearSummary,
     }
   }
 
+  private val notCodedOutPoa: TaxYearSummaryChargeItem => Boolean = chargeItem => {
```

Comment:
if we can move this under ChargeItem and add some tests around this logic?

Reply:
TaxYearSummaryChargeItem is a different type to ChargeItem, but I can move both of these bits of code into the relevant type